Sources/BraveWallet/Settings/CustomNetworkDetailsView.swift

```diff
@@ -260,17 +260,20 @@ enum CustomNetworkError: LocalizedError, Identifiable {
 struct CustomNetworkDetailsView: View {
   @ObservedObject var networkStore: NetworkStore
   @ObservedObject var model: CustomNetworkModel
+  var onCustomNetworkUpdated: (() -> Void)?
```

Comment:
This seems to just be used to call `updateAssets()` in `CryptoStore`? You should be able to use `onNetworkListChanged()` observer method for this so you don't have to pass this closure around 🙂.

Reply:
omg i was looking for an observer method for this but counln't find any cuz i was only looking at the JsonRPCServiceObserver lol
updated in [3ae6668](https://github.com/brave/brave-ios/pull/7625/commits/3ae66680a0580698e1bdd31a74d0ce0faf705cb9)